### PR TITLE
No longer publishing to artifactory

### PR DESCRIPTION
**Pull request by rcausey**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,37 +9,16 @@ jobs:
...
-    steps:
-      - checkout
-      - run: curl -ks -u$ARTIFACTORY_USER:$ARTIFACTORY_PASS "https://d2lartifacts.artifactoryonline.com/d2lartifacts/api/npm/npm-local/auth/d2l" -o .npmrc
-      - run: sudo npm install -g publish
-      - run: publish
-
 workflows:
   version: 2
 
   build-and-publish:
```
Comment (qwangd2l) [inline .circleci/config.yml]: Maybe just "build"

**Comment by qwangd2l** [comment]
Actually, is circleci process even needed anymore?
Can we just remove the whole process?

**Comment by rcausey** [comment]
> Actually, is circleci process even needed anymore?
> Can we just remove the whole process?

It's needed to ensure tests/linting passes

**Comment by qwangd2l** [comment]
> > Actually, is circleci process even needed anymore?
> > Can we just remove the whole process?
> 
> It's needed to ensure tests/linting passes

Ah that's right. :D

**Pull request merged**